README.md

```diff
@@ -148,7 +148,10 @@ docker run -it -v $HOME/prysm:/data -p 4000:4000 --name beacon-node \
 3. To run the beacon node, issue the following command:
 
 ```text
-docker run -it -v c:/prysm/:/data -p 4000:4000 gcr.io/prysmaticlabs/prysm/beacon-chain:latest --datadir=/data --clear-db
+docker run -it -v c:/prysm/:/data -p 4000:4000 -p 13000:13000 --name beacon-node \
```

Comment:
Please revert this command with the `\` for windows, as those slashes will not work for windows users. This is why we had the whole command on one line.

Reply:
I don't know much about Windows, but it makes sense.
Reverted